platforms/hyperledger-indy/configuration/samples/network-indyv3.yaml

```diff
@@ -27,6 +27,7 @@ network:
     - organization:
       name: authority
       type: peer
+      retry_count: 10
```

Comment:
Refer to corda/fabric sample. The retrycount is taken from the network.env as it is considered an environment issue and not per organization. Also it is not related to the Peer as such hence kept as a env setting.

Reply:
updated